homework_6/task_3.py

```diff
@@ -0,0 +1,10 @@
+def is_prime(arg: int):
```

Comment:
please read about prime numbers. Your code does not find these numbers. Also range is 2->1000